server_py/flatgov/bills/templatetags/bill_filters.py

```diff
@@ -39,10 +39,10 @@ def billnumber_display(billnumber: str):
         return billnumber
 
 @register.filter
-def billnumbers_display(billnumbers: list):
+def billnumbers_display(billnumbers: list, current_bill: str):
```

Comment:
Instead of adding a parameter here (since this filter is used elsewhere), I'll add a separate filter to filter billnumbers by Congress